numpythia/src/_libnumpythia.pyx

```diff
@@ -62,6 +63,16 @@ PARENTS = HepMC.PARENTS
 CHILDREN = HepMC.CHILDREN
 SIBLINGS = HepMC.PRODUCTION_SIBLINGS
 
+cdef class sphericity:
+    cdef Sphericity sph
```

Comment:
```suggestion
    cdef *cSphericity this
```

Reply:
(Then the dealloc below will work)

Reply:
When I do this, I get the error "Expected an identifier, found *", is there any other place where I need to change this definition?

Thanks!